class.jetpack.php

```diff
@@ -1330,24 +1216,6 @@ public static function refresh_theme_data() {
 	}
 
 	/**
-	 * Invalides the transient as well as triggers the update of the mock option.
-	 *
-	 * @return null
-	 */
-	function is_single_user_site_invalidate() {
-		/**
-		 * Fires when a user is added or removed from a site.
-		 * Determines if the site is a single user site.
-		 *
-		 * @since 3.4.0
-		 *
-		 * @param string jetpack_single_user_site.
-		 * @param bool Jetpack::is_single_user_site() Is the current site a single user site.
-		 */
-		do_action( 'update_option_jetpack_single_user_site', 'jetpack_single_user_site', (bool) Jetpack::is_single_user_site() );
```

Comment:
Since you're deprecating hooks, I'd recommend adding them to this function:
https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L6290